package.json

```diff
@@ -8,11 +8,12 @@
   "scripts": {
     "build:docs": "webpack && eleventy --input=./docs --output=public",
     "build:package": "node bin/check-nvmrc.js && gulp build:package",
-    "ci:testsass": "node-sass -q test.scss >/dev/null && echo 'ok'",
     "ci:release": "HUSKY_SKIP_HOOKS=1 CI=true semantic-release --debug",
     "ci:dryrun": "HUSKY_SKIP_HOOKS=1 CI=true semantic-release --dry-run",
     "start": "npm-run-all --parallel watch:*",
-    "test": "npm run ci:testsass",
+    "test": "npm-run-all --parallel test:*",
```

Comment:
So are these still manual to run or is there another file somewhere controlling the CI pipeline to run these?

Reply:
oh lawd ignore, I see what you are doing, very clever!